### ISSUE TITLE
README clarification suggestion

### ISSUE THREAD

**Issue by amy-keibler** [yananhub/flying-gradle-plugin#1]
I'm reaching out on behalf of the Maven Central team to hopefully clear up some confusion. We noticed that your README mentions

> According to Sonatype, from March 12th, 2024, all repositories will be published through the Central Portal. However, there is currently no official Gradle plugin to do this.

We think the March 12th date might be a confusion from our [news announcement about decommissioning `issues.sonatype.org`](https://central.sonatype.org/news/20240301_changes_to_account_management/), the service we used for user registration and issue tracking. While we would prefer users begin the process of migrating to publish via https://central.sonatype.com, we do not yet require that. Once we have a date publicly available, we will do our best to communicate it broadly. Given that, we would suggest rewording the README to something like

> According to Sonatype, the Central Portal will replace the legacy OSSRH service for publishing. However, there is currently no official Gradle plugin to do this.

Hopefully this suggestion is received as it is intended, as a way to help reduce confusion on what is required of publishers. Thank you for creating a Gradle plugin to publish via our service.

**Comment by yananhub** [comment]
Hi @amy-keibler ,

Thanks for your suggestion! I've updated the README per it.
